certified-connectors/Vineforce/apiDefinition.swagger.json

```diff
@@ -0,0 +1,204 @@
+                  "title": "To Email"
+                },
+                "title": {
+                  "type": "string",
+                  "description": "Title for the task.",
+                  "title": "Title"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "Description for the task",
+                  "title": "Description",
+                  "format": "html"
+                },
+                "dueDate": {
+                  "type": "string",
+                  "description": "Due Date of the task.",
+                  "title": "Due Date"
+                },
+                "resourceAppName": {
+                  "type": "string",
+                  "description": "App Name",
+                  "title": "Resource App",
+                  "enum": [
+                    "Microsoft to do's",
+                    "Office 365 Outlook",
+                    "SharePoint/SharePoint list",
+                    "OneDrive",
+                    "Adobe Cloud",
+                    "Adobe Sign",
+                    "Bitbucket",
+                    "Docusign",
+                    "Microsoft Dynamics 365",
+                    "HelloSign",
+                    "Microsoft Dataverse",
+                    "Microsoft Forms",
+                    "Teams",
+                    "Planner",
+                    "Power Automate",
+                    "Twitter",
+                    "Typeform",
+                    "Zendesk",
+                    "Jira",
+                    "Azure DevOps",
+                    "Asana"
+                  ]
+                },
+                "resourceAppUrl": {
+                  "type": "string",
+                  "description": "External link associated with the resource app",
+                  "title": "Resource Url"
+                },
+                "projectName": {
+                  "type": "string",
+                  "description": "Project to be associated with the task",
+                  "title": "Project Name"
+                },
+                "projectSectionName": {
+                  "type": "string",
+                  "description": "Project Section to be associated with the task",
+                  "title": "Project Section"
+                },
+                "checkList": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "Title of checklist item.",
+                    "title": "Checklist"
+                  },
+                  "description": "checkList"
+                }
+              },
+              "required": [
+                "apiKey",
+                "title",
+                "toEmail"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/api/services/app/ExternalTask/PushNotificationFromExternal": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@nikunjvf Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

Reply:
Updated with latest commit